capstone/src/main/java/com/google/univiz/api/MapsResource.java

```diff
@@ -1,12 +1,13 @@
 package com.google.univiz.api;
 
+import com.google.univiz.CollegeId;
 import java.util.List;
 
 /**
  * The MapResource interface is responsible for all data management/processing specific for the maps
```

Comment:
nit - keep the Javadoc up to date :)
```suggestion
 * The MapsResource interface is responsible for all data management/processing specific for the maps
```